util/__init__.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import absolute_import, division, print_function
+
+import math
+
+from libtbx.math_utils import round2
+
+
+def format_float_with_standard_uncertainty(value, standard_uncertainty, minimum=1e-15):
```

Comment:
Ah, to actually fix the behaviour this intends to address, I think we should use something like this.
```suggestion
def format_float_with_standard_uncertainty(value, standard_uncertainty, minimum=1e-12):
```